Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
+    rspec (3.8.0)
```

Comment:
### [rspec](http://github.com/rspec)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.6.0 → 3.8.0

[_~~change-log~~, [source-code](http://github.com/rspec/rspec)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/rspec/rspec/compare/v3.6.0...v3.8.0).

These are the individual commits:
- (dcb407a) [Bump version to 3.7.0.pre](https://github.com/rspec/rspec/commit/dcb407aa8b072c8c1cb612e1eeb99b9b754fa115)
- (f77250c) [Releases 3.7.0](https://github.com/rspec/rspec/commit/f77250cb331ece2c7965ad1db3e2f2416c1081bc)
- (6a96d5e) [Bump version to 3.8.pre](https://github.com/rspec/rspec/commit/6a96d5e2e27d89eb78e08d6c9b8023f9521edc64)
- (d3c155f) [Update README.md](https://github.com/rspec/rspec/commit/d3c155f60ecc63efd6ba33e1a441bd69cbcb53aa)
- (149aac2) [Merge pull request #23 from Davidslv/patch-1](https://github.com/rspec/rspec/commit/149aac2fc44b178553e4ba4a004bf9fa026efa27)
- (ea13175) [Fix a weird problem with readme](https://github.com/rspec/rspec/commit/ea13175c42c089367b7a63e60b04a0c8235f95e2)
- (ab3c913) [Merge pull request #24 from LewisVo/patch-1](https://github.com/rspec/rspec/commit/ab3c9131c4869a928732f65355524a27fc3425ca)
- (b3fa23d) [Release 3.8.0.](https://github.com/rspec/rspec/commit/b3fa23d24b0cd929a2c98f09e46977af0458e237)


</details>